egs/wsj/s5/steps/data/reverberate_data_dir.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/env python
+# Copyright 2016  Tom Ko
+# Apache 2.0
+# script to generate reverberated data
+
+# we're using python 3.x style print but want it to work in python 2.x,
+from __future__ import print_function
+import argparse, glob, math, os, random, sys, warnings, copy, imp, ast
+
+data_lib = imp.load_source('dml', 'steps/data/data_dir_manipulation_lib.py')
+
+def GetArgs():
+    # we add required arguments as named arguments for readability
+    parser = argparse.ArgumentParser(description="Reverberate the data directory with an option "
+                                                 "to add isotropic and point source noises. "
+                                                 "Usage: reverberate_data_dir.py [options...] <in-data-dir> <out-data-dir> "
+                                                 "E.g. reverberate_data_dir.py --rir-list-file rir_list "
+                                                 "--foreground-snrs 20:10:15:5:0 --background-snrs 20:10:15:5:0 "
+                                                 "--noise-list-file noise_list --speech-rvb-probability 1 --num-replications 2 "
+                                                 "--random-seed 1 data/train data/train_rvb",
+                                     formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument("--rir-list-file", type=str, required = True, 
+                        help="RIR information file, the format of the file is "
+                        "--rir-id <string,required> --room-id <string,required> "
+                        "--receiver-position-id <string,optional> --source-position-id <string,optional> "
+                        "--rt-60 <float,optional> --drr <float, optional> <location(support Kaldi IO strings)> "
```

Comment:
make similar changes specifying what is supported at all other location specifiers.